tools/kura_inspector/src/main.rs

```diff
@@ -21,27 +29,95 @@ enum Command {
         /// Number of the blocks to print.
         /// The excess will be truncated
         #[clap(short = 'n', long, default_value_t = 1)]
-        length: usize,
+        length: u64,
     },
 }
 
-#[tokio::main]
-#[allow(clippy::use_debug, clippy::print_stderr)]
-async fn main() {
+#[allow(clippy::use_debug, clippy::print_stderr, clippy::panic)]
+fn main() {
     let args = Args::parse();
-    let mut output = DefaultOutput::new();
-    Config::from(args)
-        .run(&mut output)
-        .await
-        .unwrap_or_else(|e| eprintln!("{:?}", e))
-}
 
-impl From<Args> for Config {
-    fn from(src: Args) -> Self {
-        let Args { from, command } = src;
+    let from_height = match args.from {
+        Some(height) => {
+            assert!(height != 0, "The genesis block has the height 1. Therefore the \"from height\" you specify must not be 0.");
+            // Kura starts counting blocks from 0 like an array while the outside world counts the first block as number 1.
+            Some(height - 1)
+        }
+        None => None,
+    };
 
-        match command {
-            Command::Print { length } => Config::Print(print::Config { from, length }),
+    match args.command {
+        Command::Print { length } => {
+            print_blockchain(&args.path_to_block_store, from_height, length)
         }
     }
 }
+
+#[allow(
+    clippy::print_stdout,
+    clippy::use_debug,
+    clippy::expect_used,
+    clippy::unwrap_used
+)]
+fn print_blockchain(block_store_path: &Path, from_height: Option<u64>, block_count: u64) {
```

Comment:
```suggestion
fn print_blockchain(block_store_path: &Path, from_height: u64, block_count: u64) {
```

Explicitly asking the user to specify `u64::MAX` is better and shorter.